modules/custom/activity_send/modules/activity_send_email/src/EmailFrequencyBase.php

```diff
@@ -39,7 +38,7 @@ public function getInterval() {
   /**
    * {@inheritdoc}
    */
-  public function processItem(Activity $activity, Message $message, User $target) {
+  public function processItem(Activity $activity, $body_text, User $target) {
```

Comment:
Unfortunately this is BC breaking, imagine there are already Queue items created (not finished) when people are updating their platform. 
The item being processed still has a message instead, so we need to think about how we can make it backwards compatible.

For example by adding a new optional variable `$body_text = NULL` so only when the body text is filled we can skip the entire rendering of the `$message` if it isnt we still do it the old way.